packages/core/src/locales/zh-CN.schema.yml

```
@@ -0,0 +1,87 @@
+      autoCreateRoomFromUser: 是否为每个用户自动创建专属房间。
+      defaultChatMode: 设置默认的聊天模式。
+      defaultModel: 设置默认使用的聊天模型。
+      defaultPreset: 设置默认使用的聊天预设。
+
+    - $description: 杂项
+      authSystem: 是否启用配额组和用户权限系统（实验性功能）。启用后，各适配器设置的调用限额将失效。
+      errorTemplate: 设置错误提示消息的模板（此设置在未来版本中可能会有变更）。
+      voiceSpeakId: 设置使用 vits 服务时的默认发音人 ID。
+      isLog: 是否启用调试模式。
+      isProxy:
+        $desc: 是否启用全局代理。启用后，ChatLuna 全家桶插件的网络请求将使用代理。
```

yamllint

[warning] wrong indentation: expected 10 but found 8

(indentation)